HIP/hip-904-frictionless-airdrop.md

```diff
@@ -0,0 +1,706 @@
+3. As a receiver who claims a token airdrop I will only be responsible for the transaction submission fee. No custom fees will be assessed to me as part of this transfer. 
+4. As a user with receiver signature required enabled any airdrops sent to me will result in a pending transfer, even with open slots.
+
+### General User Stories
+
+1. As an hts-token-holder I want to be able to reject unwanted tokens (regardless of how they were obtained) by sending them back to the treasury account and not having to pay any custom fees.
+2. As a non-hedera-centric wallet user, I want to obtain an auto-created account (by EVM address or public key) that is able to receive unsolicited tokens. I understand this account will have `maxAutomaticAssociations` set to -1 implying unlimited automatic associations.
+3. As account owner for an account created by an EVM tool my experience with airdrops on Hedera is identical to that on Ethereum
+4. As a network user I want to be able to accurately query the mirror node for all pending transactions sent to an account
+5. As a network user I want to be able to accurately query the mirror node for all outstanding airdrops sent by an account
+6. As a wallet user I want my wallet to show me all pending transactions to my account
+7. As an explorer user I want my explorer to show me search or view all pending transactions and outstanding  to my accounts
+8. As a DApp developer I want to expose airdrop HAPI functionality and pending transfers state information to my users
+9. As a HAPI developer I want to be able to query the cost of an airdrop transactions in HBAR
+10. As a DApp developer I want to be able to query the cost of an airdrop transactions in gas
+11. As an account that has open auto association slots, I only pay for the fees and rent for used association slots. I no longer pay for unused association slots.
+
+### FAQ
+
+- Q: I am sending a crypto transfer, but I don’t want to pay for auto-association. What can I do?
+    
+    A: Each transaction has a `transactionFee` which represents the maximum transaction fee the user is willing to pay for that transaction. If it is set low enough, then the transaction will fail instead of transferring the token in the event an association was required. The user still has to pay node and network fees for the transaction, but the transfer itself will not go through and the client will not be charged for the crypto transfer itself.
+- Q: If I have `maxAutomaticTokenAssociations` set to `0`, and someone airdrops me an NFT, have I received it?
+
+    A: No. In this case, it is *pending*, and you can only receive it by sending a “claim” transaction. When you claim it, the record stream will contain a claim transaction and a token association record entry and a crypto transfer record entry, so that in the record stream, the association and token transfer are both expressed as they normally would be.
```

Comment:
This answer, though correct, by diving into the record stream doesn't seem to match the question!  Like, TMI!  On the _other_ hand it _is_ the kind of _specification_ that does belong in this HIP - if it isn't just obvious. 

IOW, if it needs to be mentioned at all, there should be a section of this HIP that talks about the effects of these changes on the record stream.  E.g., what does a airdrop that goes to pending look like?  Is there anything special about a cancel or reject?  Etc.

Reply:
There's a record file section in the HAPI section where I could move this to.